monero-wallet-generator.html

```diff
@@ -10938,14 +10933,23 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
 </footer>
 
 <script>
+
+if (document.getElementById("currencyDropDownList").value == "Monero") {
+  var cnUtil = cnUtilGen(moneroConfig);
+  current_lang='english';
+}
+else if (document.getElementById("currencyDropDownList").value == "Aeon") {
+  var cnUtil = cnUtilGen(aeonConfig);
+  current_lang='electrum';
```

Comment:
Nit: Please add spaces on either side of the `=`

Reply:
Done.